src/main/java/de/tum/in/www1/artemis/web/rest/TextExerciseResource.java

```diff
@@ -402,7 +402,7 @@ else if (!authCheckService.isAtLeastTeachingAssistantForExercise(optionalTextExe
                     result.setFeedbacks(assessments);
                 }
 
-                if (!authCheckService.isAtLeastInstructorForExercise(textExercise, user)) {
+                if (!authCheckService.isAtLeastTeachingAssistantForExercise(textExercise, user)) {
```

Comment:
could this change be required for other exercise types as well?
it seems to work just fine, but weird that this check exists just for one type

Reply:
There is some code duplication in that regard but fixing it is not the scope of the PR.
If was correct for some occurrences and wrong for these.
I basically searched for "setAssessor(null)" in the whole project and checked the 6(+-) occurrences.

Reply:
just as expected then